Makefile

```diff
@@ -120,15 +80,10 @@ run-etcd:
 	--advertise-client-urls "http://$(LOCAL_IP_ENV):2379,http://127.0.0.1:2379" \
 	--listen-client-urls "http://0.0.0.0:2379"
 
-semaphore:
-	# Ensure Semaphore has loaded the required modules
-	modprobe -a ip6_tables xt_set
-
-	# Run the STs
-	make st
-
+semaphore: test-containerized
 	set -e; \
 	if [ -z $$PULL_REQUEST_NUMBER ]; then \
+		$(MAKE) libnetwork-plugin.created; \
```

Comment:
I agree that the .created files are ugly and we should look for places where we don't need them. As long as the docker (re)build is quick then it's fine to get rid of them, but sometimes the build can't be cached or is slow.